plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/AdvancedConnectionEditor.java

```diff
@@ -276,6 +291,17 @@ protected GridLayoutFactory adjustAdvancedCompositeLayout(GridLayoutFactory grid
 		return composite;
 	}
 
+	private void discoverRegistryPressed() {
+		String ret = RegistryProviderModel.getDefault().getRegistryURL(pageModel.getSelectedConnection());
+		if( ret == null ) {
+			System.out.println("Nothing found");
```

Comment:
Could be removed

Reply:
@robstryker nice with extension point et all. Can you please add badly needed unit test?